component/otelcol/exporter/debug/debug.go

```diff
@@ -0,0 +1,72 @@
+package debug
+
+import (
+	"github.com/grafana/agent/component"
+	"github.com/grafana/agent/component/otelcol"
+	"github.com/grafana/agent/component/otelcol/exporter"
+	otelcomponent "go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configtelemetry"
+	debugexporter "go.opentelemetry.io/collector/exporter/debugexporter"
+	otelextension "go.opentelemetry.io/collector/extension"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name:    "otelcol.exporter.debug",
+		Args:    Arguments{},
+		Exports: otelcol.ConsumerExports{},
+
+		Build: func(opts component.Options, args component.Arguments) (component.Component, error) {
+			fact := debugexporter.NewFactory()
+			return exporter.New(opts, fact, args.(Arguments), exporter.TypeAll)
+		},
+	})
+}
+
+// Arguments configures the otelcol.exporter.debug component.
+type Arguments struct {
+	Verbosity          configtelemetry.Level `river:"verbosity,attr,optional"`
+	SamplingInitial    int                   `river:"sampling_initial,attr,optional"`
+	SamplingThereafter int                   `river:"sampling_thereafter,attr,optional"`
+
+	// DebugMetrics configures component internal metrics. Optional.
+	DebugMetrics otelcol.DebugMetricsArguments `river:"debug_metrics,block,optional"`
```

Comment:
I think we don't actually need `debug_metrics` in  `otelcol.exporter.debug`. It's ok to remove it. It is only really useful for exporter/receiver which send/receive data over the network. 

`otelcol.exporter.logging` has it, but I think it was unnecessary that we'd added it there. We can leave it in `otelcol.exporter.logging`, but it would help keep `otelcol.exporter.debug` cleaner and less confusing if we don't have it.